File: Cartas.Web/Domain/Models/Player.cs
namespace Cartas.Web.Domain.Models
{
    public class Player
    {
        public string PlayerId { get; set; }
        public string PlayerName { get; set; }
        public string Avatar { get; set; }
        public int Seat { get; set; }
        public int WinCount { get; set; }
    }
}

File: Cartas.Web/Scripts_ts/game/player.ts
class Player {
    playerId: string;
    playerName: string;
    avatar: string;
    seat: number;
    winCount: number;

    constructor(playerId: string, playerName: string, avatar: string, seat: number, winCount: number) {
        this.playerId = playerId;
        this.playerName = playerName;
        this.avatar = avatar;
        this.seat = seat;
        this.winCount = winCount;
    } 
}